diagnostics/TUTT/doc/TUTT.rst

```diff
@@ -0,0 +1,120 @@
+
+(**) cropping.py can be referenced if code is needed to either shift the grid of your data
+or to crop your data to a specified region
+
+As a module of the MDTF code package, all scripts of this package can be found under
+``mdtf/MDTF_$ver/diagnostics/TUTT
+
+Required programming language and libraries
+-------------------------------------------
+
+Python3 packages: "netCDF4", "skimage", "numpy", "scipy", "shapely.geometry", "cartopy"
+
+Required model input variables
+-------------------------------
+
+Time-varying 2-D geopotential height fields at 200 hPa (unit: gpm). 200-hPa geopotneitla height should be the monthly-mean field. Horizontal resolution of the geopotential height data can be deceided by the user. Note that the longitude index of the data must goes from west to east (0-360E).
+
+
+References
+----------
+
+.. _ref-Muñoz1:
+
+Wang, Z., Zhang, G., Dunkerton, T. J., & Jin, F. F. (2020). Summertime stationary waves integrate tropical and extratropical impacts on tropical cyclone activity. Proceedings of the National Academy of Sciences of the United States of America, 117(37), 22720-22726. https://doi.org/10.1073/pnas.2010547117
+
+Chuan-Chieh Chang and Zhuo's paper is under developing...
+
+
+
+More about this diagnostic
+--------------------------
+
+a. Weak westerly wind sometimes extends southward of the equator and connects to the westerlies over the Southern Hemisphere in reanalysis datasets. We therefore choose to use the zonal geostrophic wind with a fixed Coriolis parameter instead of the total zonal wind.
+
+b. Pacific TUTT and Atlantic TUTT sometimes are connected to each other. This has been observed during the years when the 200-hPa anticyclone over the Central America is weak. The longitude used to divide two TUTTs can be decided by the following options:
+
+    1. The script calculates the averaged 200-hPa geopotential height between 20 to 30 N, and then searches the longitude where the averaged 200-hPa geopotential height is the largest between 120 W to 80 W.   This option uses the approximate location of subtropical ridge over the North America as the dividing longitude.
+    2. The dividing longitude is specified by the user.
+
+.. figure:: TUTT_example.png
+   :align: center
+   :width: 75 %
+   
+
+   Figure 1. Solid gray curve denotes the postions where the climatological seasonal-mean Ug equals to 2 m/s in JRA-55 reanalysis from 1958 to 2014. Dashed white line indicates the reference latitude. Estimated Pacific TUTT and Atlantic TUTT indices are shown above the figure. Background shaded field is climatolgoical 200-hPa geopotential height (gpm). 
+   
+
+Descriptions of outputted .txt files:
+
+TUTT_contour_lat_1958-Ug_2.0.txt/TUTT_contour_lon_1958-Ug_2.0.txt
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The corrdinates (latitude and longitude) of TUTT contour given by 200 hPa zonal geostrophic wind (Ug) at value 2 m/s.
```

Comment:
The ~~corrdinates~~ coordinates of the TUTT contour given by the 200 hPa zonal geostrophic wind (Ug) at a value of 2 m/s.

Reply:
Revised. Thank you.